docs/backend-system/building-backends/08-migrating.md

```diff
@@ -754,3 +754,255 @@ going easily, but feel free to move it out to where it fits best. As you migrate
 your entire plugin flora to the new backend system, you will probably make more
 and more of these modules as "first class" things, living right next to the
 implementations that they represent, and being exported from there.
+
+### The Auth Plugin
+
+A basic installation of the auth plugin will look as follows.
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-auth-backend'));
+backend.add(import('@backstage/plugin-auth-backend-module-microsoft-provider'));
+/* highlight-add-end */
+```
+
+An additional step you'll need to take is to add the resolvers to your configuration, here's an example:
+
+```yaml title:"app-config.yaml
+auth:
+  environment: development
+  providers:
+    microsoft:
+      development:
+        clientId: ${AZURE_CLIENT_ID}
+        clientSecret: ${AZURE_CLIENT_SECRET}
+        tenantId: ${AZURE_TENANT_ID}
+        signIn:
+          resolvers:
+            - resolver: emailMatchingUserEntityAnnotation
+            - resolver: emailMatchingUserEntityProfileEmail
+            - resolver: emailLocalPartMatchingUserEntityName
+```
+
+> Note: the resolvers will be tried in order, but will only be skipped if they throw a `NotFoundError`.
+
+#### Auth Plugin Modules and Their Resolvers
+
+As you may have noticed in the above example you'll need to import the `auth-backend` and an `auth-backend-module`. The following sections outline each of them and their resolvers.
+
+All of the following modules include the following common resolvers:
+
+- [emailMatchingUserEntityProfileEmail](https://github.com/backstage/backstage/blob/5447cffd23cf00772988fb799ced0ec5e54efb2e/plugins/auth-node/src/sign-in/commonSignInResolvers.ts#L29)
```

Comment:
could we link all these to the microsite instead? eg https://backstage.io/docs/reference/plugin-auth-node.commonsigninresolvers.emailmatchinguserentityprofileemail/

Reply:
Yeah was thinking the same, although the presentation of these in the API reference isn't great. Thinking it might also be a bit tricky to handle this as we move to community plugin repo etc and might need to find some other solutions for package API references. We could probably merge as is as I'm not really sure a link to the API reference is better in this case?